src/Model/Message.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace App\Model;
+
+use Symfony\Component\Validator\Constraints as Assert;
+
+class Message
+{
+    public const TYPE_SMS = 'sms';
```

Comment:
This also violates Open Closed principle and this type of object polymorphism can go to wrong direction easily